python/boto3/security/hardcoded-token.yaml

```diff
@@ -0,0 +1,22 @@
+rules:
+- id: hardcoded-token
+  patterns:
+  - pattern-either:
+    - pattern: $W(..., aws_secret_access_key="=~/[A-Za-z0-9/+=]+/", ...)
```

Comment:
Would a `+{16,}` remove your additional false positives?

Reply:
I don't think the Ocaml Regex supports {N}... 😞 
cf. https://caml.inria.fr/pub/docs/manual-ocaml/libref/Str.html

Reply:
I guess that isn't a thing in OCAML regex, so you'd have to write the character class 16 times :(

Reply:
I did try it with {40} and I think it was interpreted as a literal `{40}`

Reply:
cf. https://github.com/returntocorp/semgrep/issues/688